### PR TITLE
Jarv/fab updates

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: package_data.yaml [hunk 1]
```diff
@@ -134,76 +134,97 @@ post_checkout_regex: !!omap
...
+        if [[ -e /opt/wwc/edx-platform/requirements/edx-sandbox/local.txt ]]; then
+          cd /opt/wwc/edx-platform
+          sudo -E /opt/edx-sandbox/bin/pip install -q --upgrade --no-deps --ignore-installed \
+          --exists-action w -r /opt/wwc/edx-platform/requirements/edx-sandbox/local.txt
+          fi
+        sudo /usr/sbin/aa-enforce /etc/apparmor.d/code.sandbox
+      fi
   - ^edx-platform$:
     - |
+      # On servers running the workers, which run from the same cloned
```
Comment (jbau) [inline package_data.yaml:160]: curious what the worker upstart script looks like - do you mean worker in terms of celery?  If so, would love to have a copy.
@e0d or @jarv

**Comment by jarv** [comment]
@jbau we are thinking that keeping this fork will be difficult to maintain until we are ready to start using it.  Are you able to deploy using the fab scripts in the sysadmin repo instead?  We may want to do that for now.

**Comment by jbau** [comment]
I'm game to try.  I assume this is mitx/sysadmin?  Also, can I get access to mitx/configuration too?

**Comment by jarv** [comment]
Sure thing.
I was also thinking it might be helpful to give you access to our stage
environment. Can you paste your public key.

**Comment by jbau** [comment]
```
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDEf0jEsH18/cjTVaiDHSTTcJw62t1klsgkbLeCb+YgH+gvirHGladp/GqW0PXwaYvBSDFZkdZxuXgqgrtvWDUZit0OIv3dsh7oaFg0w10dDKUwwaGsWH7cKytfzvBXdeuVQUGTSQWRnaeKEjxL3m71A+nmWnS071hliN9BZN4H8jSpGdpgF+wPQGSCMpZZBeiwq6lbsUnyI5wezuCGsGeJmTxyVUZRC9JWmL5C71xFKtFZ9IFilxR+AVd6ppFPVQiH8A5x68Qo2HN3UpkuTn42/oS+KOXTTdEOPFnQ5Qup1sGXDlv3K0Lg5UPlWQ3/2wpAmuS93aJr9lQ3G8zM4gJJ jbau@stanford.edu
```

**Comment by jbau** [comment]
BTW, found this fun fact:

http://thechangelog.com/github-exposes-public-ssh-keys-for-its-users/

**Comment by jbau** [comment]
I just ran a release with this fab file, so I'll go ahead and merge it in.

**Pull request merged**